brianlong/www.validators.app
# 178677228 validators on a map

#### What's this PR do?
- Adds validators view on a map

#### How should this be manually tested?
- populate your db with some data - daemons and:
```
  rails r script/update_gossip_nodes.rb
  rails r script/data_centers_scripts/append_data_centers_geo_data.rb
  rails r "DataCenterStatsWorker.perform_async('mainnet')"
  rails r "DataCenterStatsWorker.perform_async('testnet')"
```
- make sure you have your assets pre-compiled locally (if you have errors, run `rm -r node_modules public/packs` and `yarn install`) and run server
-  go to localhost:3000/validators-map and test

#### What are the relevant tickets?
- URL to the PivotalTracker ticket

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
